sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs

```diff
@@ -15,7 +15,6 @@ namespace Azure.DigitalTwins.Core.Tests
     /// This class will initialize all the settings and create and instance of the Digital twins client.
     /// </summary>
     [Parallelizable(ParallelScope.Self)]
-    [Category("Live")] // This category indicates the tests hit a "live" service: https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#to-test-1
```

Comment:
Not needed anymore. The value will be set based on the Environment variable in the pipeline and user config locally.
Ex:
{
    "DigitalTwinsInstanceHostName": "instance",
    "ApplicationId": "appid",
    "ClientSecret": "secret",
    **"TestMode":  "Live"**
}